virttest/cartesian_config.py

```diff
@@ -1856,55 +1908,6 @@ def _parse(self, lexer, node=None, prev_indent=-1):
                                          lexer.line))
             raise
 
-    def drop_suffixes(self, d):
```

Comment:
Not sure if this public(?) method is no longer used, maybe you could confirm with the author @Andrei-Stepanov

Reply:
Let's see what he has to say but considering the very low usage of this code and the fact that getting a feedback for a simple thing might take months, I would consider hour long silence on his side as acknowledgement.